autowarefoundation/autoware
# Feature/release debians 766

## Status
**PRODUCTION**

## Description
I fixed 5 more pkgs so they are releasable. Also fixed issue with Qt5::Core being needed even though the package doesn't use it (for map_file pkg only, in other pkgs we still need to fix it).

## Todos


## Steps to Test or Reproduce
`./catkin_make_release`

At this point it would be great if we would have a standard sequence of launch files that we execute and at least visually check if everything still works. 
@yk-fujii @amc-nu @YamatoAndo do you guys have any procedure like this?

I will assign all authors of the pkgs.

## Issue
autowarefoundation/autoware_ai#960
